app/graphql/types/invoices/object.rb

```diff
@@ -46,6 +46,12 @@ class Object < Types::BaseObject
       field :fees, [Types::Fees::Object], null: true
       field :invoice_subscriptions, [Types::InvoiceSubscription::Object]
       field :subscriptions, [Types::Subscriptions::Object]
+
+      # NOTE(legacy): Remove with taxes feature
+      field :sub_total_vat_excluded_amount_cents, GraphQL::Types::BigInt, null: false, method: :sub_total_excluding_taxes_amount_cents
```

Comment:
Layout/LineLength: Line is too long. [134/120] (https://rubystyle.guide#max-line-length)